src/pages/community.js

```diff
@@ -0,0 +1,683 @@
+              projects are looking for passionate people to work for them..
+            </Subtitle>
+            <ButtonRow>
+              <ButtonLink to="/community/grants/">
+                {/* <Translation id="page-community-poap-button" /> */}
+                Explore Grants
+              </ButtonLink>
+              <StyledButtonLink
+                isSecondary
+                to="/community/get-involved/#ethereum-jobs/"
+              >
+                {/* <Translation id="page-community-poap-secondary-button" /> */}
+                Find a job
+              </StyledButtonLink>
+            </ButtonRow>
+          </FeatureContent>
+          <ImageContainer>
+            <FeatureImage
+              fluid={data.ethereum.childImageSharp.fluid}
+              alt={translateMessageId(
+                "page-community-open-source-image-alt",
+                intl
+              )}
+            />
+          </ImageContainer>
+        </RowReverse>
+      </OpenSourceContainer>
+      <PoapContainer>
+        <Row>
+          <FeatureContent>
+            <LeftColumnContent>
+              <H2>
+                {/* <Translation id="page-community-poap" /> */}
+                Contribute to ethereum.org
+              </H2>
+              <Subtitle>
+                {/* <Translation id="page-community-poap-description" /> */}
+                For many people, ethereum.org is their first step into the
+                ecosystem. It is kept up-to-date and accurate by thousands of
+                open-source contributors. Want to help? Read our guide on
+                contributing, or take up an issue on our GitHub.
+              </Subtitle>
+              <ButtonRow>
+                <ButtonLink to="/contributing/">
+                  {/* <Translation id="page-community-poap-button" /> */}
+                  More on contributing
+                </ButtonLink>
+                <StyledButtonLink
+                  isSecondary
+                  to="https://github.com/ethereum/ethereum-org-website/"
+                >
+                  {/* <Translation id="page-community-poap-secondary-button" /> */}
+                  View on GitHub
+                </StyledButtonLink>
+              </ButtonRow>
+            </LeftColumnContent>
+          </FeatureContent>
+          <ImageContainer>
+            <FeatureImage
+              fluid={data.finance.childImageSharp.fluid}
+              alt={translateMessageId("page-index-internet-image-alt", intl)}
+            />
+          </ImageContainer>
+        </Row>
+      </PoapContainer>
+      <SupportContainer>
+        <RowReverse>
+          <FeatureContent>
+            <H2>
+              {/* <Translation id="page-community-support" /> */}
+              Developer support
+            </H2>
+            <Subtitle>
+              {/* <Translation id="page-community-support-description" /> */}
+              The Ethereum community has adopted many standards that are helpful
```

Comment:
This feels very specific 🤔  - I think a dedicated section to general "Ethereum support" would be the way to go vs. getting into e.g. EIPs or specifically support for devs. I think we can cover that on the support page.

Reply:
This was placeholder from the original designs. I'm not actually sure what we should put here (or if we want a 3rd secondary call to action). I've left it in for now just incase to save writing it twice.